### DISCUSSION TITLE
CR10 S5 Profile Request

### DISCUSSION THREAD

**Discussion by CWCZX3** [OrcaSlicer/OrcaSlicer#3597] [Dev]
Possible to get this added?

**Comment by jsilvius** [comment]
CR-10, CR-10 S4, CR-10 S5 would all be great.

**Comment by Nihilus77** [comment]
I know this is old but The S5 would be great especially funning Klipper

**Comment by Will-wastelander** [comment]
Any updates on this ? Just installed OrcaSlicer 2.3.1, and there is no profile for the CR10 S5.
